frontends/web/src/routes/account/info/info.tsx

```diff
@@ -0,0 +1,140 @@
+import { route } from 'preact-router';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { getInfo, IAccount, ISigningConfigurationList } from '../../../api/account';
+import { isBitcoinBased } from '../utils';
+import { ButtonLink } from '../../../components/forms';
+import { Header } from '../../../components/layout';
+import * as style from './info.css';
+import { SigningConfiguration } from './signingconfiguration';
+import { BitcoinBasedAccountInfoGuide } from './guide';
+
+interface InfoProps {
+    accounts: IAccount[];
+    code: string;
+}
+
+interface State {
+    info?: ISigningConfigurationList;
+    viewXPub: number;
+}
+
+type Props = InfoProps & TranslateProps;
+
+class Info extends Component<Props, State> {
+    public readonly state: State = {
+        info: undefined,
+        viewXPub: 0,
+    }
+
+    public componentDidMount() {
+        getInfo(this.props.code).then(info => this.setState({ info }));
+    }
+
+    public componentWillMount() {
+        document.addEventListener('keydown', this.handleKeyDown);
+    }
+
+    public componentWillUnmount() {
+        document.removeEventListener('keydown', this.handleKeyDown);
+    }
+
+    private handleKeyDown = (e: KeyboardEvent) => {
+        if (e.keyCode === 27) {
+            route(`/account/${this.props.code}`);
+        }
+    }
+
+    private getAccount(): IAccount | undefined {
+        if (!this.props.accounts) {
+            return;
+        }
+        return this.props.accounts.find(({ code }) => code === this.props.code);
+    }
+
+    private showNextXPub = (): void => {
+        if (!this.state.info) {
+            return;
+        }
+        const numberOfXPubs = this.state.info.signingConfigurations.length;
+        this.setState(({ viewXPub }) => ({
+            viewXPub: (viewXPub + 1) % numberOfXPubs
+        }));
+    }
+
+    private xpubTypes = ['p2wpkh', 'p2wpkh-p2sh', 'p2pkh']
+
+    public render(
+        { t, code }: RenderableProps<Props>,
+        { info, viewXPub }: State
+    ) {
+        const account = this.getAccount();
+        if (!account || !info) return null;
+        const config = info.signingConfigurations[viewXPub];
+        const numberOfXPubs = info.signingConfigurations.length;
+        return (
+            <div class="contentWithGuide">
+                <div class="container">
+                    <Header title={<h2>{t('accountInfo.title')}</h2>} />
+                    <div class="innerContainer scrollableContainer">
+                        <div class="content padded">
+                            <div class="box larger">
+                                { isBitcoinBased(account.coinCode) ? (
+                                    <h2 className={style.title}>
+                                        {t('accountInfo.extendedPublicKey')}
+                                    </h2>
+                                ) : null }
+                                { numberOfXPubs > 1 ? (
```

Comment:
maybe ` && config.bitcoinSimple`? then the `?` below on line 106 maybe is not needed, if the compiler is smart enough.

Reply:
No I'd never use `&&` for conditional rendering, reason: if `numberOfXPubs` is the number 0 then JSX would actually render `0` and you'd want to render nothing.

Reply:
>  if numberOfXPubs is the number 0 then JSX would actually render 0 and you'd want to render nothing.

that would be with `||` not with`&&`?

Reply:
Updated had to change to `{ (config.bitcoinSimple !== undefined && numberOfXPubs > 1) ? (` to make TypeScript happy, PTAL